cookbook/validation/validation-group-service-resolver.rst

```diff
@@ -0,0 +1,55 @@
+How to dynamically configure validation groups
+=============================================
+
+If you need some advanced logic to determine the validation groups that cannot
+be simply determined by a simple callback (for example that requires a few 
+services) you can use a service. Implement a service that implements ``__invoke``
```

Comment:
The first sentence is a very long one. What about "Sometimes you need advanced logic to determine the validation groups. If this can't be determined by a simple callback, you can use a service."